### ISSUE TITLE
JavaScript Errors on latest 0.7.x

### ISSUE THREAD

**Issue by zetas** [NodeBB/NodeBB#3167]
I already posted this as a bug on your forums so here's the details:

https://community.nodebb.org/topic/4548/javascript-errors-in-latest-0-7-x

As of right now i can't use my ACP :-1: 

I should know better than to just do a pull but I was having issues and thought it might help.

**Comment by lots0logs** [comment]
This worked for me recently:
```
rm -rf public/src
rm public/nodebb.min.js
git checkout .
./nodebb upgrade
```

**Comment by zetas** [comment]
Yea I did that exact thing already haha. I'm pretty good at debugging stuff :+1: good suggestion though.

**Comment by julianlam** [comment]
@zetas, can you please paste the error message from the browser console? Is it something along the lines of `Unexpected identifier`?

**Comment by julianlam** [comment]
Oh, I didn't realise you were the same as the person in that topic.

Can you try something for me? Go into your ACP and move all of your active widgets to the draft zone. Save, and go see if the error has disappeared.

**Comment by datagutt** [comment]
I am having the same error. Tried the fix but it doesn't work.
I can't log into ACP as i get "Forbidden" (csrf) error.

**Comment by datagutt** [comment]
OK, seems to be my theme (?)

**Comment by zetas** [comment]
@julianlam Moving the widgets to draft did nothing. I continued messing with it for a long time, even re-installing fresh with 0.7.x and still got the errors in chrome however in safari i didn't. 

I went and disabled all chrome extensions just in case and still no dice. I'm still seeing the javascript errors in the ACP in chrome. Here's a screenshot of what the ACP looks like with the inspector open:

![image](https://cloud.githubusercontent.com/assets/1036810/7941545/fd69a682-0925-11e5-9332-59517aab334c.png)
